macOS/LocalPackages/DataBrokerProtection-macOS/Sources/DataBrokerProtection-macOS/Debug/DebugScanJob.swift

```diff
@@ -172,6 +174,14 @@ final class DebugScanJob: SubJobWebRunning {
         await executeNextStep()
     }
 
+    func canPerformNextAction(_ action: Action) async -> Bool {
+        if action.actionType == .expectation {
+            retriesCountOnError = 1
```

Comment:
I'm a little wary of the function name - from the name I would assume this to have no side effects, simply checking some state to return a boolean, but it's modifying the retry count. Since this is a debug menu option though, I think it's fine overall.

Reply:
Yeah I don’t like the current name either, but have no better idea. Any suggestions?

Reply:
Renamed to `evaluateActionAndHaltIfNeeded` as discussed over MM.